vipps-order-management-api.md

```diff
@@ -93,17 +93,18 @@ As the same OrderId can be used for both a Recurring charge and a Ecom payment,
 
 ### Basic flow
 
-1. Initiate a Vipps eCom or recurring payment
-    - `POST:/ecomm/v2/payments`
-2. Save the **orderId** you used when initiating
+1. Initiate a Vipps eCom API payment or a Vipps Recurring API charge:
+  [`POST:/ecomm/v2/payments`](https://vippsas.github.io/vipps-developer-docs/api/ecom#tag/Vipps-eCom-API/operation/initiatePaymentV3UsingPOST)
+  or
+  [https://vippsas.github.io/vipps-developer-docs/api/recurring#tag/Charge-v2-endpoints/operation/CreateCharge]
+2. Save the `orderId` you used when initiating.
 3. Add an image (optional)
-    - `POST:/v1/images`
-4. Save the imageId
+  [`POST:/order-management/v1/images`](https://vippsas.github.io/vipps-developer-docs/api/order-management#tag/Image/operation/postImage)
+4. Save the `imageId`
 5. Add a category with link and image
-    - `/v2/{paymentType}/categories/{orderId}`
+  [`POST:/order-management/v1/{paymentType}/categories/{orderId}`](https://vippsas.github.io/vipps-developer-docs/api/order-management#tag/Category/operation/putCategoryV2)
 6. Add a receipt
-    - `/v2/{paymentType}/receipts/{orderId}`
-
+  [`POST:/order-management/v1/{paymentType}/receipts/{orderId}`](https://vippsas.github.io/vipps-developer-docs/api/order-management#tag/Receipt/operation/postReceiptV2)
```

Comment:
change to:
  [`POST:/order-management/v2/{paymentType}/receipts/{orderId}`](https://vippsas.github.io/vipps-developer-docs/api/order-management#tag/Receipt/operation/postReceiptV2)